From 90fb6f288d32f033bd8355d317c5580b1c836775 Mon Sep 17 00:00:00 2001
From: Chase Douglas <chase.douglas@canonical.com>
Date: Mon, 13 Sep 2010 21:48:24 +0200
Subject: [PATCH 1/1] Set default input module to evdev for Apple Magic Mouse (LP: #637430)

* Set default input module to evdev for Apple Magic Mouse (LP: #637430)
  - Add debian/local/60-magictrackpad.conf
---
 debian/changelog                   |    7 +++++++
 debian/local/60-magictrackpad.conf |    5 +++++
 debian/rules                       |    4 ++++
 3 files changed, 16 insertions(+), 0 deletions(-)
 create mode 100644 debian/local/60-magictrackpad.conf

diff --git a/debian/changelog b/debian/changelog
index 225c61f..b502133 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+xserver-xorg-input-evdev (1:2.3.2-6ubuntu2) maverick; urgency=low
+
+  * Set default input module to evdev for Apple Magic Mouse (LP: #637430)
+    - Add debian/local/60-magictrackpad.conf
+
+ -- Chase Douglas <chase.douglas@ubuntu.com>  Mon, 13 Sep 2010 21:46:44 +0200
+
 xserver-xorg-input-evdev (1:2.3.2-6ubuntu1) maverick; urgency=low
 
   * Add gesture support
diff --git a/debian/local/60-magictrackpad.conf b/debian/local/60-magictrackpad.conf
new file mode 100644
index 0000000..4b7f4aa
--- /dev/null
+++ b/debian/local/60-magictrackpad.conf
@@ -0,0 +1,5 @@
+Section "InputClass"
+         Identifier "Magic Trackpad"
+         MatchUSBID "05ac:030e"
+         Driver "evdev"
+EndSection
diff --git a/debian/rules b/debian/rules
index cddc86e..d9e4e1c 100755
--- a/debian/rules
+++ b/debian/rules
@@ -21,6 +21,7 @@ ifneq (,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
 endif
 
 DEB_HOST_ARCH      ?= $(shell dpkg-architecture -qDEB_HOST_ARCH)
+DEB_HOST_ARCH_OS   ?= $(shell dpkg-architecture -qDEB_HOST_ARCH_OS)
 DEB_HOST_GNU_TYPE  ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 DEB_BUILD_GNU_TYPE ?= $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
 ifeq ($(DEB_BUILD_GNU_TYPE), $(DEB_HOST_GNU_TYPE))
@@ -84,6 +85,9 @@ binary-arch: build install serverabi
 	dh_installdocs
 	dh_installchangelogs ChangeLog
 	dh_install --sourcedir=debian/tmp --list-missing --exclude=evdev_drv.la
+ifeq ($(DEB_HOST_ARCH_OS), linux)
+	install -D -m 644 debian/local/60-magictrackpad.conf debian/xserver-xorg-input-evdev/usr/share/X11/xorg.conf.d/60-magictrackpad.conf
+endif
 	dh_installman
 	dh_link
 	dh_strip
-- 
1.7.1

